resteasy-client-microprofile-base/src/main/java/org/jboss/resteasy/microprofile/client/ProxyInvocationHandler.java

```diff
@@ -151,6 +151,8 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
                 }
                 if (cause instanceof ExceptionMapping.HandlerException) {
                     ((ExceptionMapping.HandlerException)cause).mapException(method);
+                    // no applicable exception mapper found or applicable mapper returned null
+                    return null;
```

Comment:
I'm not sure yet what the right answer is, but I'm not sure `null` should be returned here. Since this is invoking a proxy on an interface returning `null` here may break the interface contract. I don't think we can make assumptions on the implementation.

Reply:
Please see response in following section.